src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -791,10 +791,6 @@ private Configurations.Units GetTransformedRevitUnits(Configurations.Units revit
             {
                 return Configurations.Units.Millimeters;
             }
-            else if (revitUnits == Configurations.Units.Inches)
```

Comment:
Is this not needed?

Reply:
Actually, it turns out that it is not needed. For Imperial units, the direct translation of scale works great. For Metric units, an inverse translation had to take place for things to look 'normal' against the grid. I, then, removed the unnecessary translation for Inches.